PowerShell Scanners/User AD Groups/User AD Groups.ps1

```diff
@@ -5,7 +5,13 @@ $ds = [System.DirectoryServices.ActiveDirectory.Forest]::GetCurrentForest().Find
 # Add in the desired memberof property to the query
 [void]$ds.PropertiesToLoad.Add('memberof')
 # We search by SID as while a SAM name is unique per domain it's not unique in a forest but the SID is unique in a forest
-$ds.Filter = 'objectsid={0}' -f $account.LastLoggedOnUserSID
+$lastuser = $account.LastLoggedOnUser -replace '\.\\', "\$env:COMPUTERNAME"
```

Comment:
```suggestion
$lastuser = $account.LastLoggedOnUser -replace '\.\\', "$env:COMPUTERNAME\"
```

Reply:
Without the change above, you get `\COMPUTERNAMEUsername` as the user that it is trying to find the groups for. 
Looks like even with this change it is possible to get the message `Cannot index into a null array.` These both appear to be issues when the user found is a local user instead of a domain user.